README.md

```diff
@@ -1,6 +1,33 @@
 # Snorkeling
 
-A pilot project to extract medical indications (when a compound treats a disease) from the literature using [Snorkel](https://github.com/HazyResearch/snorkel).
+This repository stores data and code to scale up the extraction of biomedical relationships (i.e. Disease-Gene assocaitions, Compounds binding to Genes, Gene-Gene interactions etc.) from the Pubmed Abstracts.
+
+This work uses a subset of [Hetionet v1](https://doi.org/cdfk), which is a heterogenous network that contains pharmacological and biological information in the form of nodes and edges. 
+(Resource metagraph shown in figure below).
```

Comment:
What do you mean by "resource metagraph"?  Do you mean just an example of a hetergeneous network?  Though it looks like you're highlighting certain paths here.  Might need to add a bit more detail on what the figure is showing.

Reply:
resource metagraph -> database schema. It shows the overall content that is contained in the database. I updated the readme for clarity.